.github/workflows/Nightly-Image.yml

```diff
@@ -1,28 +1,49 @@
-name: Nightly-Image-Build
-
-defaults:
-  run:
-    shell: bash
-
+name: Build Nightly Image
 on:
-  workflow_call:
+  schedule:
+    # Planifier l'exécution à minuit tous les jours (00:00)
```

Comment:
@SKB-TECH can you please translate to english ? Same as on line 10

Reply:
> @SKB-TECH can you please translate to english ? Same as on line 10

Right away and sorry about that